source/includes/fact-legacy-connections.rst

```diff
@@ -0,0 +1,8 @@
+Starting in version 1.39.2, |compass| no longer supports  migration from legacy 
```

Comment:
[question]

Should "migration" be "migrating" ? Changing to a verb might make this a bit clearer with active voice.

Reply:
Ah you're right. Missed that change because I had to rephrase some things 😓  Updated!